### PR TITLE
[DEVOPS-428] fixing alarms for RDS in aws prod

### PR DESCRIPTION
**Pull request by myrabokhari**
##### Corresponding Issue(s):
 - PRs should have a corresponding issue. If no issue exists, provide details in the **Reason for Change(s)** section

##### Reason for Change(s):

- If a bug, describe error scenario, including expected behavior and actual behavior.

- If an enhancement, describe the use case and the perceived benefit(s).

##### Note to the PR requester about Closing PR's
Please message the person that opened the issue when auto closing it on slack, as well as any other stake holders of deep interest. Only close the issue if you believe that the issue is fully resolved with this PR.

### CODE REVIEW

**Comment by myrabokhari** [comment]
@gotrevor-notarize this is for fixing the current plan in aws prod TF. 
Some changes to sync with the current rds alarms, so that TF gives no changes in TF plan. Should be good to go!

File: main.tf [hunk 1]
```diff
@@ -343,7 +343,7 @@ module "replica_lag_alarm_ticket" {
 }
 
 module "free_storage_space_alarm_email" {
-  source = "git@github.com:/notarize/terraform-aws-cloudwatch_alarm.git?ref=v.0.0.1"
+  source = "git@github.com:/notarize/terraform-aws-cloudwatch_alarm.git?ref=v.0.0.2"
```
Comment (gotrevor-notarize) [inline main.tf:346]: Is there a reason to only go up to 0.0.2 here, when you go to 0.0.6 below, in the same module?

**Review by gotrevor-notarize — CHANGES_REQUESTED** [review]
Clarification on why you include different revisions of the same module.

**Comment by myrabokhari** [comment]
> Clarification on why you include different revisions of the same module.

it actually depends on the details/configuration sof the version. In some alarms the old version sync with their current configuration and with some alarms it doesn’t (it shows changes in the plan). 
So to sync all the alarms correctly I had to do that.

**Review by gotrevor-notarize — APPROVED** [review]
Looking forward to a path toward using the latest version of various module. For now, :shipit:

**Pull request merged**